src/blockchain/milestones.js

```diff
@@ -144,7 +143,7 @@ const milestonesFactory = app => {
         );
       }
 
-      await updateMilestoneStatus(
+      return updateMilestoneStatus(
```

Comment:
Instead of await and return nothing, I returned the promise, to can write assertion for function output in test file